src/policy/policy.cpp

```diff
@@ -37,7 +37,13 @@ CAmount GetDustThreshold(const CTxOut& txout, const CFeeRate& dustRelayFeeIn)
     if (txout.scriptPubKey.IsWitnessProgram(witnessversion, witnessprogram)) {
         // sum the sizes of the parts of a transaction input
         // with 75% segwit discount applied to the script size.
-        nSize += (32 + 4 + 1 + (107 / WITNESS_SCALE_FACTOR) + 4);
+        if (witnessversion == 0) {
+            // P2WPKH spend, a compressed public key + a DER-encoded ECDSA signature
+            nSize += (32 + 4 + 1 + (107 / WITNESS_SCALE_FACTOR) + 4);
+        } else {
+            // Taproot key spend, a single BIP341 signature
+            nSize += (32 + 4 + 1 + (65 / WITNESS_SCALE_FACTOR) + 4);
```

Comment:
The outcome is the same, but I think the witness size is 66 (1 byte 0x01 to indicate the stack size, 1 byte 0x40 to indicate the size of the first element, 64 byte sig).

Reply:
Thanks, forgot the stack size byte